crates/cli/src/model/datadog_api.rs

```diff
@@ -258,6 +344,24 @@ pub struct SecretRuleApiType {
     pub attributes: SecretRuleApiAttributes,
 }
 
+impl From<SecretRuleApiType> for SecretRule {
+    fn from(val: SecretRuleApiType) -> Self {
+        SecretRule {
+            id: val.id.clone(),
+            name: val.attributes.name.clone(),
+            description: val.attributes.description.clone(),
+            pattern: val.attributes.pattern.clone(),
+            default_included_keywords: val
+                .attributes
+                .default_included_keywords
+                .clone()
+                .unwrap_or_default(),
+            validators: val.attributes.validators.clone(),
+            match_validation: val.attributes.match_validation.map(|v| v.into()),
```

Comment:
Like Jason said, these clones are not necessary.

```suggestion
            id: val.id,
            name: val.attributes.name,
            description: val.attributes.description,
            pattern: val.attributes.pattern,
            default_included_keywords: val
                .attributes
                .default_included_keywords
                .unwrap_or_default(),
            validators: val.attributes.validators,
            match_validation: val.attributes.match_validation.map(|v| v.into()),
```